DOC: Make pyplot import explicit in the 10 minutes to pandas page

yuseitahara opened the pull request:
Beginners don't know what is `plt`, thus it is better to show how to import it explicitly.

This is a continuation of https://github.com/pandas-dev/pandas/pull/30274

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

yuseitahara committed:
Make pyplot import explicit in the 10 minutes to pandas page

Beginners don't know what is `plt`, thus it is better to show how to import it explicitly.

yuseitahara commented:
@datapythonista Could you please merge this PR? Thank you.

datapythonista reviewed (approved):
Lgtm, thanks!

jreback commented on doc/source/getting_started/10min.rst:
```diff
@@ -717,6 +711,8 @@ of the columns with labels:
 
 .. ipython:: python
 
+   import matplotlib.pyplot as plt
```
I would leave this where it was, and maybe even add a comment (removing the suppress is ok)

yuseitahara replied:
@jreback Thank you for your comment. I moved the import statement to the previous position and added a comment.

yuseitahara committed:
import pyplot at the beginning of Plotting section

Also add a comment about the standard convention for referencing the matplotlib API.

yuseitahara commented:
> AttributeError: module 'IPython.utils.py3compat' has no attribute 'cast_bytes_py2'

pandas-dev.pandas (Web_and_Docs) is failing, but is it related to my change?

https://dev.azure.com/pandas-dev/pandas/_build/results?buildId=24127&view=logs&j=5291ec4e-9543-503b-43ce-895cf0e06ba3&t=ba0a2b35-e1f8-54df-fd66-2b9fde6f4b1f&l=5158

rushabh-v commented:
>pandas-dev.pandas (Web_and_Docs) is failing, but is it related to my change?

I also have the same problem in my PR. see #30282

datapythonista commented:
@yuseitahara can you merge master into your branch please? It should fix the CI.

datapythonista commented:
@jreback you're changes have been addressed, all green here.

jreback commented:
thanks @yuseitahara
